### DISCUSSION TITLE
⛔ Elementor Pro changelog RRS feed accessibility

### DISCUSSION THREAD

**Discussion by pauserratgutierrez** [elementor/elementor#18509] [Feature Request]
Hi everyone.

I'm searching for a json version of the Elementor Pro changelog. I'm currently trying to have access to a RRS feed version of the Elementor Pro changelog. (This changelog can only be viewed using your official page:  https://elementor.com/pro/changelog/).

I'm trying to use wordpress rest API endpoints such as "wp-json/wp/v2/pages" but the "/pro/changelog" page endpoint isn't listed there, I can't find a way to access it.

_More context: 
I have an Elementor discord server with update notifications but currently I can't display Pro notification updates because they aren't handled with Github.
I already have access to the Free Elementor changelog (because it is public at GitHub) and GitHub has RRS feed available using atom endpoints. Accessible thru this link: https://github.com/elementor/elementor/releases.atom
As the Elementor Pro changelog is only under the official Elementor website, there's no way I can think of accessing it._

**Comment by pauserratgutierrez** [comment]
Response from Support
![image](https://user-images.githubusercontent.com/73240206/166142041-0ba4ecaa-e86e-46a5-96b9-8e137e9259d8.png)

Hope devs implement this soon @shilo-ey @avivu

**Comment by nicholaszein** [comment]
Hello @pauserratgutierrez!

We currently do not offer this functionality and have no plans to open an RSS feed for the Elementor Pro changelog for security reasons.

We hope you can understand.

Kind regards
